src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/Asn1/TimeAsn.xml.cs

```diff
@@ -53,7 +53,7 @@ internal void Encode(AsnWriter writer)
                 if (wroteValue)
                     throw new CryptographicException();
                 
-                writer.WriteGeneralizedTime(GeneralTime.Value);
+                writer.WriteGeneralizedTime(GeneralTime.Value, true);
```

Comment:
Nit: can we use parameter names so for the bools so it's clear what "true" means here?

Reply:
I thought about it, then decided it was generated code so it doesn't matter.  Since there has now been a request: done.